python/pyarrow/_cuda.pyx

```diff
@@ -185,6 +185,12 @@ cdef class Context(_Weakrefable):
             cudabuf = GetResultValue(self.context.get().Allocate(nbytes))
         return pyarrow_wrap_cudabuffer(cudabuf)
 
+    def memory_manager(self):
```

Comment:
Can you add a brief docstring for both attributes? (you can probably just copy what we added for those attributes elsewhere, e.g. on the Buffer class)

Reply:
Ah, and I would make those properties instead of methods? 
At least that's what we did on Buffer and Array. I think it makes most sense to be consistent here.

Reply:
> Can you add a brief docstring for both attributes? (you can probably just copy what we added for those attributes elsewhere, e.g. on the Buffer class)

Oh whoops! I was so excited that it worked, that I forgot to add docstrings. I will for sure. Properties makes a lot of sense.